importlib_metadata/__init__.py

```diff
@@ -29,6 +29,8 @@
 from importlib import import_module
 from importlib.abc import MetaPathFinder
 from itertools import starmap
+from pydantic import Field, validate_arguments
+from pydantic.typing import Annotated
```

Comment:
Why is this being imported from `pydantic.typing` instead of `typing`?

Reply:
That's what the [docs recommended](https://github.com/python/importlib_metadata/actions/runs/2552038722).

Reply:
Okay. I didn't find anything about it, in fact, the snippets I saw from the docs were importing it from `typing_extensions`, that's why I was confused.

Reply:
Oops. That was the wrong link (now corrected).